src/ontology/cl-edit.owl

```diff
@@ -21529,29 +21530,32 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_0004220 "https
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0004220 "cell")
 AnnotationAssertion(oboInOwl:id obo:CL_0004220 "CL:0004220")
 AnnotationAssertion(rdfs:label obo:CL_0004220 "flag amacrine cell")
-SubClassOf(obo:CL_0004220 obo:CL_0000561)
+SubClassOf(obo:CL_0004220 obo:CL_0000745)
```

Comment:
@aleixpuigb, can you point me to the reference that classifies this 'flag amacrine cell' as a horizontal cell and not an amacrine cell?

Reply:
This might be a problem inherited from another PR/merging the master branch to this one. Thank you for spotting it.